src/main/test/com/epam/preprod/sirenko/strategy/impl/ManualProductCreationStrategyTest.java

```diff
@@ -0,0 +1,97 @@
+package com.epam.preprod.sirenko.strategy.impl;
+
+import com.epam.preprod.sirenko.enums.PetGroup;
+import com.epam.preprod.sirenko.enums.Season;
+import com.epam.preprod.sirenko.enums.Size;
+import com.epam.preprod.sirenko.util.ConsoleReader;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.mockito.Spy;
+
+import java.io.IOException;
+import java.io.PrintStream;
+import java.math.BigDecimal;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+class ManualProductCreationStrategyTest {
+	private PrintStream outMock;
+	
+	@InjectMocks
+	@Spy
+	private ManualProductCreationStrategy manualProductCreationStrategy;
+	
+	@Mock
+	private ConsoleReader consoleReader;
+	
+	@BeforeEach
+	public void SetData() {
+		outMock = mock(PrintStream.class);
+		System.setOut(outMock);
+		MockitoAnnotations.initMocks(this);
+	}
+	
+	@Test
+	void getPetGroupShouldReturnPetGroup() throws IOException {
+		doNothing().when(manualProductCreationStrategy).printPetGroupElements();
+		doNothing().when(outMock).println();
```

Comment:
doNothing().when(outMock).println(); is duplicated in each scenario

Reply:
made private method